drivers/interrupt_controller/intc_gicv3.c

```diff
@@ -274,6 +274,17 @@ void gic_raise_sgi(unsigned int sgi_id, uint64_t target_aff,
  */
 static void gicv3_rdist_enable(mem_addr_t rdist)
 {
+#if defined(CONFIG_GIC_V3_600)
+	uint32_t pwrr = sys_read32(rdist + GICR_PWRR);
```

Comment:
nit: we might use `GICD_IIDR >= 0x2` to indicate there is a need to set `GICR_PWRR`. Because GIC 700 also requires setting `GICR_PWRR`